src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.hpp

```diff
@@ -23,14 +23,9 @@
  *
  */
 
-
-#ifndef SHARE_GC_SHENANDOAH_SHENANDOAHSCANREMEMBERED_HPP
-#define SHARE_GC_SHENANDOAH_SHENANDOAHSCANREMEMBERED_HPP
-
-#include <stdint.h>
-#include "memory/iterator.hpp"
-#include "gc/shenandoah/shenandoahCardTable.hpp"
-
+// During development of this new feature, we want the option to test
+// with and without, and to compare performance before and after.
+#define FAST_REMEMBERED_SET_SCANNING
```

Comment:
You'll still want the normal #ifndef SHARE_GC_SHENANDOAH_SHENANDOAHSCANREMEMBERED_HPP protection, don't you?

Reply:
Guards were further down in the file, I pulled them up closer to where you'd expect them.